frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -16,16 +15,20 @@ type StrategyItemContainerProps = {
     onDragStart?: DragEventHandler<HTMLButtonElement>;
     onDragEnd?: DragEventHandler<HTMLButtonElement>;
     headerItemsRight?: ReactNode;
+    headerItemsLeft?: ReactNode;
```

Comment:
This is where this becomes relevant. Because they're styled differently from the right header items, I think it makes sense for them to handled as a separate prop. Also, as evidenced here, they're not all actions, so "headerActions" etc isn't that accurate a name.